### ISSUE TITLE
NTR: Abuse of sympathomimetic drugs (Triggers or responses: drugs or treatments)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9511]
New term request Abuse of sympathomimetic drugs (nadea3_211012200817): Cocaine, ecstasy, uppers  UMLS: 0154536

**Comment by pnrobinson** [comment]
@MickeySegal
I do not think the label is unambiguous, because presumably it does not want to include non-recreational sympathomimetic drugs?

**Comment by MickeySegal** [comment]
I'd be OK with something like "High use" or "use of high doses".

**Comment by pnrobinson** [comment]
@MickeySegal 
This is similar to this existing term [Addictive stimulant use HP:0033512](https://hpo.jax.org/browse/term/HP:0033512)

**Comment by MickeySegal** [comment]
OK

**Comment by pnrobinson** [comment]
closing
